meta.go

```diff
@@ -174,19 +174,19 @@ func (r Relation) IsImplicit() bool {
 // Meta represents all the known content that may be defined
 // within a charm's metadata.yaml file.
 type Meta struct {
-	Name            string              `bson:"name"`
-	Summary         string              `bson:"summary"`
-	Description     string              `bson:"description"`
-	Subordinate     bool                `bson:"subordinate"`
-	Provides        map[string]Relation `bson:"provides,omitempty"`
-	Requires        map[string]Relation `bson:"requires,omitempty"`
-	Peers           map[string]Relation `bson:"peers,omitempty"`
-	Format          int                 `bson:"format,omitempty"`
-	OldRevision     int                 `bson:"oldrevision,omitempty"` // Obsolete
-	Categories      []string            `bson:"categories,omitempty"`
-	Tags            []string            `bson:"tags,omitempty"`
-	SupportedSeries []string            `bson:"supported-series,omitempty"`
-	Storage         map[string]Storage  `bson:"storage,omitempty"`
+	Name        string              `bson:"name" json:"name"`
+	Summary     string              `bson:"summary" json:"summary"`
+	Description string              `bson:"description" json:"description"`
+	Subordinate bool                `bson:"subordinate" json:"subordinate"`
+	Provides    map[string]Relation `bson:"provides,omitempty" json:"provides,omitempty"`
+	Requires    map[string]Relation `bson:"requires,omitempty" json:"requires,omitempty"`
+	Peers       map[string]Relation `bson:"peers,omitempty" json:"peers,omitempty"`
+	Format      int                 `bson:"format,omitempty" json:"format,omitempty"`
+	OldRevision int                 `bson:"oldrevision,omitempty"` // Obsolete
+	Categories  []string            `bson:"categories,omitempty" json:"categories,omitempty"`
+	Tags        []string            `bson:"tags,omitempty" json:"tag,omitempty"`
+	Series      []string            `bson:"series,omitempty" json:"supported-series,omitempty"`
```

Comment:
Perhaps add a comment here:

// Series is serialised for backward compatibility
// as "supported-series" because a previous
// charm version had an incompatible Series field that
// was unused in practice but still serialized. This
// only applies to JSON because Meta has a custom
// YAML marshaler.

?